app/src/pages/inside/profilePage/accessTokenBlock/accessTokenBlock.scss

```diff
@@ -25,6 +25,36 @@
   white-space: nowrap;
   text-overflow: ellipsis;
 }
+
+.token-value-block {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  padding: 6px 10px;
+  border: 1px solid $COLOR--gray-80;
+  font-size: 13px;
+  line-height: 13px;
+  color: $COLOR--charcoal-grey;
+}
+
+.copy {
+  flex-grow: 0;
+  flex-shrink: 0;
+  height: 16px;
+  width: 16px;
+  margin-left: 10px;
+  padding: 0;
+  outline: none;
+  border: 0;
+  background: none;
+  cursor: pointer;
+  vertical-align: middle;
+  stroke: $COLOR--gray-60;
+  &:hover {
```

Comment:
Please recheck all styles and delete unused props f.e flex-grow, vertical-align...